library/src/main/java/com/smartmobilefactory/epubreader/EpubScrollDirection.java

```diff
@@ -1,7 +1,45 @@
 package com.smartmobilefactory.epubreader;
 
 public enum EpubScrollDirection {
+
+    /**
+     * make chapters internally vertical scrollable
+     * but each chapter has its own page
+     * the user can swipe horizontally threw the chapters
+     *
+     * | chapter1 | | chapter2 | | chapter3 | | chapter4 |
+     * | chapter1 | | chapter2 | | chapter3 | | chapter4 |
+     * | chapter1 | | chapter2 |              | chapter4 |
+     * | chapter1 |
+     * | chapter1 |
+     */
     HORIZONTAL_WITH_VERTICAL_CONTENT,
+
+    /**
+     * make all chapters vertical scrollable
+     * WARNING: this mode is not well tested and experimental
+     *
+     * | chapter1 |
+     * | chapter1 |
+     * | chapter1 |
+     * | chapter1 |
+     * | chapter1 |
+     * | chapter2 |
+     * | chapter2 |
+     * | chapter2 |
+     * | chapter3 |
+     * | chapter4 |
+     */
     VERTICAL_WITH_VERTICAL_CONTENT,
+
+    /**
+     * only show one chapters which vertical scrollable
```

Comment:
Only a comment, but strange grammar ;). Did you mean "show only a single chapter which is vertically scrollable"?